jobs/JOBSPROC_RAP_PREP

```diff
@@ -115,7 +120,7 @@ export SENDDBN=${SENDDBN:-YES}
 # Paths for obsproc items
 # -------------------------------
 export HOMEobsproc=${HOMEobsproc:\
--${NWROOT:?}/obsproc.${obsproc_ver:?\
+-${PACKAGEROOT:?}/obsproc.${obsproc_ver:?\
```

Comment:
See same as JOBSPROC_RAP_DUMP*. Please change to:
export HOMEobsproc=${HOMEobsproc:\
-${PACKAGEROOT:?}/${obsNET}/${obsproc_ver:?\